examples/basic/basic_pysdk_demo_video_stream.ipynb

```diff
@@ -0,0 +1,155 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "5c15cb24",
+   "metadata": {},
+   "source": [
+    "![Degirum banner](https://raw.githubusercontent.com/DeGirum/PySDKExamples/main/images/degirum_banner.png)\n",
+    "## AI Inference on a video stream\n",
+    "This notebook is a simple example of how to use DeGirum PySDK to do AI inference on a video stream.\n",
+    "\n",
+    "This script works with the following inference options:\n",
+    "\n",
+    "1. Run inference on DeGirum Cloud Platform;\n",
+    "2. Run inference on DeGirum AI Server deployed on a localhost or on some computer in your LAN or VPN;\n",
+    "3. Run inference on DeGirum ORCA accelerator directly installed on your computer.\n",
+    "\n",
+    "To try different options, you need to specify the appropriate `hw_location` option. \n",
+    "\n",
+    "You also need to specify your cloud API access token in `degirum_cloud_token`.\n",
+    "\n",
+    "You can change `video_source` to index of a local webcamera, or URL of an RTSP stream, or URL of a YouTube video, or path to another video file.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "76681f07",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# make sure degirum-tools package is installed\n",
+    "!pip show degirum || pip install degirum"
```

Comment:
As above, no degirum_tools here.

Also, I know I'm paranoid, but I really don't like this block in general. First code I see in the sample installs something weird, and doesn't even explain what it is. Also, for samples where degirum-tools is used, do we know if pip via notebook has good error handling? What does it say if something goes wrong?

Reply:
@AlexABolotov  Done